### PR TITLE
borgbackup: make logs a bit cleaner if local archive is used

### PR DESCRIPTION
**Pull request by szaimen**
Follow-up to https://github.com/nextcloud/all-in-one/pull/4804

Prevent logs like the following when used with local archive:
```
cat: can't open '.pub': No such file or directory
Your public ssh key for borgbackup is: 
```

### CODE REVIEW

File: Containers/borgbackup/backupscript.sh [hunk 1]
```diff
@@ -66,7 +66,7 @@ if [ -n "$BORG_REMOTE_REPO" ] && ! [ -f "$BORGBACKUP_KEY" ]; then
     ssh-keygen  -f "$BORGBACKUP_KEY" -N ""
     echo "You should configure the remote to accept this public key"
 fi
-if [ -n "$BORG_REMOTE_REPO" ]
+if [ -n "$BORG_REMOTE_REPO" ] && [ -f "$BORGBACKUP_KEY" ]; then
```
Comment (szaimen) [inline Containers/borgbackup/backupscript.sh:69]: @timdiels does this make sense?

Comment (szaimen) [reply Containers/borgbackup/backupscript.sh]: okay, figured it out 🙈

**Comment by timdiels** [comment]
Nit: I would merge both ifs

- if remote
  - if key does not exist, create it 
  - if key exists, cat it

**Comment by szaimen** [comment]
> Nit: I would merge both ifs
> 
> * if remote
>   
>   * if key does not exist, create it
>   * if key exists, cat it

I see... Do you want to create a PR? :)

**Comment by timdiels** [comment]
No no, either is fine really

**Pull request merged**